### PR TITLE
config and utility functions

### PR DESCRIPTION
**Pull request by kevqyzhu**

### CODE REVIEW

File: dnadoh/data_config.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+DNA: "ACGT"
+SNP_PROBS: !!python/tuple [0.50, 0.25, 0.13, 0.12]
```
Comment (tammy-dg) [inline dnadoh/data_config.yml:2]: this is cool, is this some .yml specific syntax for defining types and defaults?

**Comment by tammy-dg** [comment]
Hey Kevin, nice work with the .yml! I wonder if it'd be more straightforward to add these parameters to `example-parameter.json`(but then I guess they'd have to end up as CLI options in the argparse too, and this doesn't really become a config file anymore). I'm really not sure - does @gvwilson have any wisdom to share?

File: dnadoh/data_config.yml (hunk 1 above)
Comment (kevqyzhu) [reply dnadoh/data_config.yml:2]: When parsing yaml, I believe it automatically defaults to lists, so you need to specify that the objects are tuples. It's a little bit clunky, so we can try thinking of a more elegant solution!

**Comment by kevqyzhu** [comment]
Closing this as there is no real point right now to include a separate config file for some of the parameters that are in data_config.yml; these can be included in example-parameter.json for now. We can figure out long-term parameter documentation later on.

**Pull request closed without merging**